fireant/slicer/widgets/highcharts.py

```diff
@@ -181,6 +184,13 @@ def group_series(keys):
             colors, series_colors = itertools.tee(colors)
             axis_color = next(colors) if 1 < total_num_items else None
 
+            if isinstance(axis, self.PieChart):
+                # pie charts suck
```

Comment:
Understand that, but I'd find more valuable to know why they require such a special treatment. Or why we need the None appended to the references list bellow.

Reply:
yes i'll put a better comment